### ISSUE TITLE
conda clean --lock is not doing anything currently

### ISSUE THREAD

**Issue by gabrielcnr** [conda/conda#4903]
As of today (20 March 2017) the current implementation of `conda clean --lock` in HEAD(9d43c7e)   does not do anything at all.

I found out because I had to interrupt the `conda build` process when it was at the stage of reindexing the local channels and I noticed that the lock file got stuck inside the `noarch` local channel inside `conda-bld`.

The promise of the command is to ***remove all conda lock files***. I'm not sure which other lock files are set by conda, but I could definitely start by submitting a Pull Request that removes any lock files inside the local channels in `conda-bld` directory.

**Comment by kalefranz** [comment]
Yeah that's interesting.  We removed all locks in conda itself in 4.3, although we might bring one of them back in 4.4 or 4.5.

For the conda-build lock, mind filing an issue in the conda-build repo?  The code change would still have to be to conda, but the conda-build team knows best the details of how cleaning up conda-build locks should work.

**Comment by goanpeca** [comment]
On 4.6

Even if not doing anything now the CLI command is failing

```
$ conda clean --lock

ArgumentError: At least one removal target must be given. See 'conda clean --help'.
```

**Comment by kalefranz** [comment]
Feel free to add it back

**Comment by raybellwaves** [comment]
I came across this today

**Comment by oushujun** [comment]
Check the `..../miniconda3/pkgs/cache/` folder and remove any `*.conda_lock` files. Works for me.

**Comment by poedator** [comment]
this option disappeared entirely in conda 4.12 (wsl, ubuntu 20):
```
$ conda clean --lock
usage: conda [-h] [-V] command ...
conda: error: unrecognized arguments: --lock
```
Now, if it is deprecated, what is the alternative?

**Comment by soapy1** [comment]
Hi there,

Thanks for reporting this bug. I'm sure this was a real issue when originally reported (our fault for not looking sooner!). `conda clean` no longer supports the `--lock` argument. Conda should no longer be using these files, but if you find them on your system, you can remove them with the following `find "${path_to_folter}" -type f -name "*.conda_lock"  | xargs rm`.

Cheers!
